docs/marketplace/dev/setup/marketplace-supported-browsers.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Marketplace supported browsers
+description: This documents lists browsers which the Spryker Marketplace supports.
+last_updated: Jun 8, 2022
+template: howto-guide-template
+---
+
+This document lists browsers which the Spryker Marketplace supports.
+
+### Supported browsers
+
+The Spryker Marketplace supports the following browsers:
+
+| DESKTOP (MARKETPLACE AND MERCHANT PORTAL) | TABLET (MARKETPLACE AND MERCHANT PORTAL) | MOBILE (MARKETPLACE ONLY) |
+| --- | --- | --- |
+| *Browsers*: <ul><li> Windows, macOS: Chrome (latest version)</li> <li>Windows: Firefox (latest version)</li><li>Windows: Edge (latest version)</li><li>macOS: Safari (latest version)</li></ul> *Windows versions*:<ul><li>Windows 10</li><li>Windows 7</li></ul>*macOS versions*:<ul><li> Catalina (basically, the latest released version)</li></ul>*Screen resolutions*:<ul><li>1024-1920 width</li></ul>| *Browsers*: <ul><li>iOS: Safari</li><li>Android: Chrome</li></ul>*iOS versions*:<ul><li>iOS 13</li></ul>*Screen resolutions*:<ul><li>1024x703—for example, iPad Air</li></ul>|*Browsers*: <ul><li>iOS: Safari</li><li>Android: Chrome</li></ul>*Screen resolutions*:<ul><li>360x640—for example, Samsung Galaxy S8 or S9)</li><li>375x667—for example, iPhone 7 or 8</li><li>iPhone X, Xs, Xr</li></ul>*Android versions*:<ul><li>8.0</li></ul>*iOS versions*:<ul><li>iOS 13 (basically, the latest released version)</li></ul> |
```

Comment:
Here you have Desktop / Tablet / Mobile but on Yves/Zed page, you have Desktop / Mobile / Tablet.
We need to make the table with the same structure.